app/controllers/account/profiles_controller.rb

```diff
@@ -14,6 +14,6 @@ def update
   end
 
   def users_params
-    params.require(:user).permit(:first_name, :last_name, :role, :department, :about)
+    params.require(:user).permit(:first_name, :last_name, :role, :department, :about, :avatar)
```

Comment:
I want to see test where you are sending avatar
so before test user without it
update
and user with avatar